core/src/edu/rice/comp413fall2020grey/Location/Message/QueryResponse.java

```diff
@@ -0,0 +1,39 @@
+package edu.rice.comp413fall2020grey.Location.Message;
+
+import edu.rice.comp413fall2020grey.Common.Message;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * Message for responding to query requests.
+ * Sent from the registrar to the superpeers.
+ */
+public class QueryResponse extends Message {
+
+    /**
+     * A set of mappings for obtaining new
+     * objects of interest. Maps are organized as follows:
+     *
+     *      GameObject UUID -> Corresponding server UUID
+     */
+    private final Set<Map<UUID, UUID>> objectsOfInterest;
```

Comment:
This might get confusing in the future so maybe having a uuid wrapper class for both the super peer uuid and object uuid could clear things up

Reply:
Added ServerUUID and GameObjectUUID wrapping classes.

Reply:
Should we then change the mapping to use these new classes